packages/tooling/__tests__/operation.test.js

```diff
@@ -4,10 +4,16 @@ const petstore = require('./__fixtures__/petstore.json');
 const multipleSecurities = require('./__fixtures__/multiple-securities.json');
 const referenceSpec = require('./__fixtures__/local-link.json');
 
+describe('#getContentType', () => {
+  it('should return the content type on an operation', () => {
```

Comment:
Are there any other tests in oas-to-har that we can pull over? Few little weird edge cases in that function! 

https://github.com/readmeio/api-explorer/blob/06aa341b629680b255e1469e427c76fe08e05bc3/packages/oas-to-har/__tests__/index.test.js#L1325-L1470

Reply:
There were a couple I had missed.